examples/zephyr/common/settings_autoload.c

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2022 Golioth, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/init.h>
+#include <zephyr/settings/settings.h>
+
+static int settings_autoload(void)
+{
+	return settings_load();
```

Comment:
```suggestion
    return settings_load();
```

formatting: spaces versus tabs